sympy/physics/units/quantities.py

```diff
@@ -89,7 +89,7 @@ def _eval_is_constant(self):
         return self.scale_factor.is_constant()
 
     def _eval_Abs(self):
-        return self.__class__(self.name, self.dimension,
+        return Quantity(self.name, self.dimension,
```

Comment:
Concerning the code editing... it's usually faster to make small fixes by just editing the code rather than asking the PR author to fix the code.